File: 26_Remove_Duplicates_from_Sorted_Array/solution.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        index = 0

        for i in range(len(nums) - 1):
            if nums[i] == nums[i + 1]:
                continue
            else:
                nums[index] = nums[i]
                index += 1

        nums[index] = nums[-1]
        return index + 1

File: 26_Remove_Duplicates_from_Sorted_Array/solution.c
int removeDuplicates(int* nums, int numsSize) {
    int index = 0;

    for (int i = 0; i < numsSize - 1; i++) {
        if (nums[i] == nums[i + 1]) {
            continue;
        } else {
            nums[index++] = nums[i];
        }
    }

    nums[index] = nums[numsSize - 1];
    return index + 1; //kyuki size return krna hai
}